### PR TITLE
Simulate streaming until the actual streaming is working

### PR DESCRIPTION
**Pull request by jorisoudejans**
- Edit camera-list view to show stream mock
- Edit DatabaseSeeder to comform to the used video's
- Add video's

### CODE REVIEW

**Comment by skoppers** [comment]
Great! This will be very usefull for the presentation!

**Comment by jorisoudejans** [comment]
Oh wait, I forgot to add the music! Hold on

File: app/assets/stylesheets/director.scss [hunk 1]
```diff
@@ -147,6 +147,22 @@ $grid-padding: 4px;
...
+
+  .next {
+    width: 50%;
+    box-sizing: border-box;
+    padding-left: 5px;
+    padding-right: 5px;
+    padding-bottom: 2px;
+    padding-top: 2px;
+  }
+
```
Comment (hiddely) [inline app/assets/stylesheets/director.scss:165]: Nice CSS!

Comment (jorisoudejans) [inline app/assets/stylesheets/director.scss:165]: Thanks bruh

**Pull request merged**